doc/grpc-proxy.md

```diff
@@ -0,0 +1,51 @@
+[中文](grpc-proxy-cn.md)
+# grpc-proxy
+
+proxying gRPC traffic:
+gRPC client -> APISIX -> gRPC server
+
+### Parameters
+
+* `service_protocol": "`:  the route's option `service_protocal` must be `grpc`
```

Comment:
we can write the doc to [admin API](https://github.com/iresty/apisix/blob/master/doc/admin-api-cn.md#route).

write a simple explanation is enough.